projects/1.16/assets/pams-harvestcraft-2-food-extended/pamhc2foodextended/lang/zh_cn.json

```
@@ -104,6 +105,55 @@
   "item.pamhc2foodextended.strawberrysodaitem": "草莓汽水",
   "item.pamhc2foodextended.theatreboxitem": "影院套餐",
   "item.pamhc2foodextended.ediblerootitem": "食用块茎",
+  "item.pamhc2foodextended.baconmushroomburgeritem": "培根蘑菇汉堡",
+  "item.pamhc2foodextended.bamboosteamedriceitem": "竹蒸饭",
+  "item.pamhc2foodextended.bananasplititem": "香蕉船",
+  "item.pamhc2foodextended.biscuititem": "饼干",
+  "item.pamhc2foodextended.biscuitsandgravyitem": "肉汤饼干",
+  "item.pamhc2foodextended.bltitem": "番茄培根三明治",
+  "item.pamhc2foodextended.breadedporkchopitem": "煎面包屑包猪排",
+  "item.pamhc2foodextended.broccolimacitem": "西兰花通心粉 ",
+  "item.pamhc2foodextended.brownieitem": "榛果巧克力蛋糕",
+  "item.pamhc2foodextended.carrotjuiceitem": "胡萝卜汁",
+  "item.pamhc2foodextended.cheezepuffsitem": "芝士泡芙",
+  "item.pamhc2foodextended.cherryicecreamitem": "樱桃冰淇淋",
+  "item.pamhc2foodextended.chickennoodlesoupitem": "鸡肉面条汤",
+  "item.pamhc2foodextended.chilidogitem": "辣椒热狗",
+  "item.pamhc2foodextended.chiliitem": "辣椒",
+  "item.pamhc2foodextended.chocolatemilkitem": "巧克力牛奶",
+  "item.pamhc2foodextended.cinnamonappleoatmealitem": "肉桂苹果燕麦粥",
+  "item.pamhc2foodextended.cinnamonsugardonutitem": "肉桂甜甜圈",
+  "item.pamhc2foodextended.cinnamontoastitem": "肉桂吐司",
+  "item.pamhc2foodextended.coleslawitem": "凉拌卷心菜",
+  "item.pamhc2foodextended.cornbreaditem": "玉米面包",
+  "item.pamhc2foodextended.corndogitem": "玉米热狗",
+  "item.pamhc2foodextended.cornflakesitem": "玉米片",
+  "item.pamhc2foodextended.cornonthecobitem": "玉米棒",
+  "item.pamhc2foodextended.cranberrysauceitem": "红莓酱",
+  "item.pamhc2foodextended.crispyricepuffcerealitem": "锅巴泡芙麦片",
+  "item.pamhc2foodextended.delightedmealitem": "至尊宝套餐",
+  "item.pamhc2foodextended.deluxecheeseburgeritem": "豪华芝士汉堡",
+  "item.pamhc2foodextended.eggsaladitem": "鸡蛋沙拉",
+  "item.pamhc2foodextended.epicbltitem": "史诗级番茄培根三明治",
+  "item.pamhc2foodextended.frenchtoastitem": "法式吐司",
+  "item.pamhc2foodextended.frosteddonutitem": "糖针甜甜圈",
+  "item.pamhc2foodextended.gravyitem": "肉汤",
+  "item.pamhc2foodextended.greenbeancasseroleitem": "焗烤四季豆",
+  "item.pamhc2foodextended.groundcinnamonitem": "肉桂粉",
+  "item.pamhc2foodextended.loadedbakedpotatoitem": "猪扒起司烤土豆",
```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
  "item.pamhc2foodextended.loadedbakedpotatoitem": "盛宴烤土豆",
```
努力还原原文的韵味，若是想不出词可在群里问。